ninja.sh

```diff
@@ -0,0 +1,36 @@
+package: ninja
+version: "1.8.2"
+tag: "v1.8.2"
+source: https://github.com/ninja-build/ninja
+build_requires:
+ - "GCC-Toolchain:(?!osx)"
+prefer_system: .*
+prefer_system_check: |
+  which ninja && case `ninja --version | sed -e 's/.* //' | cut -d. -f1,2,3 | head -n1` in "^(0|2)\.|^1\.([0-7]|[9-99])\.\d*|^1\.8\.[0-1]" ) exit 1 ;; esac
+---
+#!/bin/bash -e
+
+echo "Building ALICE Ninja. To avoid this install Ninja >= 1.8.2"
+
+$SOURCEDIR/configure.py --bootstrap
+mkdir -p $INSTALLROOT/bin
+cp ./ninja $INSTALLROOT/bin
+
+# Modulefile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0
+# Our environment
+setenv NINJA_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+prepend-path PATH \$::env(NINJA_ROOT)/bin
```

Comment:
No `lib`, can you confirm? (By the way, if this is the case, it's great. It means we can really construct a system from scratch by having ninja+compiler only, and not also the dependencies to run ninja itself...)

Reply:
No lib besides system ones : 

~/alice/sw/osx_x86-64/ninja/latest/bin
❯ otool -L ninja
ninja:
        /usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 400.9.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.50.4)